Configuration.md

```diff
@@ -247,6 +249,8 @@ for what kind of requests one can make.
 
 ### MARS
 
+**Licence**: By using MARS Dataset, users agree to the terms and conditions specified in [Licence](https://cds.climate.copernicus.eu/api/v2/terms/static/licence-to-use-copernicus-products.pdf) document.
```

Comment:
I think there's a more appropriate link for the MARS license. I think this one is better: https://www.ecmwf.int/en/forecasts/accessing-forecasts/licences-available